charts/llmisvc-resources/templates/config-llm-decode-template.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
 spec:
   template:
     containers:
-      - image: ghcr.io/llm-d/llm-d:v0.2.0
+      - image: ghcr.io/llm-d/llm-d-dev:v0.2.2
```

Comment:
is this expected?

Reply:
yes, I wanted to pick up the latest to get some vllm fixes, the plan is to eventually (soon) switch to 0.3